templates/hooks/parametergroup/sdk_update_post_set_output.go.tpl

```diff
@@ -0,0 +1,13 @@
+ko, err = rm.setParameters(ctx, ko)
+
+if err != nil {
+    return nil, err
+}
+
+if delta.DifferentAt("Spec.Tags") {
+    err = rm.updateTags(ctx, ko, desired, latest);
+}
+
+if err != nil {
+    return nil, err
```

Comment:
Why we cannot do this in `sdk_update_pre_build`?

Reply:
Yes, I just changed it.